rust/lance/src/dataset/scanner.rs

```diff
@@ -338,6 +407,28 @@ impl Scanner {
         self
     }
 
+    /// Sort the results of the scan by one or more columns
+    ///
+    /// If Some, then the resulting stream will be sorted according to the given ordering.
+    /// This may increase the latency of the first result since all data must be read before
+    /// the first batch can be returned.
+    pub fn order_by(&mut self, ordering: Option<Vec<ColumnOrdering>>) -> Result<&mut Self> {
```

Comment:
Should we set as None if the vec is empty?

Reply:
Good catch, yes.